public/app/features/alerting/unified/components/contact-points/ContactPointHeader.tsx

```diff
@@ -107,10 +188,20 @@ export const ContactPointHeader = ({ contactPoint, disabled = false, onDelete }:
             {referencedByPoliciesText}
           </TextLink>
         )}
+        {numberOfRules > 0 && (
```

Comment:
Now the `unused` badge for Grafana AM contact points should be accurate 🙌 As we can now distinguish from an auto generated simpified routing policy vs actually being used directly by a rule